### ISSUE TITLE
Admin: Non-admin view is extremely basic.

### ISSUE THREAD

**Issue by beaulebens** [Automattic/jetpack#3915]
Currently non-admins see a very basic box/button combo, which is not very compelling.

![screen shot 2016-05-23 at 6 13 08 pm](https://cloud.githubusercontent.com/assets/108942/15488286/0882800c-2112-11e6-9bd9-8e3491bda9c8.png)

They should see a slimmed-down version of our pre-connection screen, targeted towards whatever features are available for them, e.g. SSO, Publicize, Stats, Calypso management.

**Comment by beaulebens** [comment]
Perhaps should be a separate issue, but we also need to figure out what to display once a user is connected. That view is also very bare right now. If they can change any settings or should be able to access any parts of the admin UI, we should be supporting that. Currently they see:

![screen shot 2016-05-23 at 6 35 31 pm](https://cloud.githubusercontent.com/assets/108942/15488591/345ec1ba-2115-11e6-8c97-0ffa258a96e0.png)

**Comment by dereksmart** [comment]
What do we show them when none of the modules that non-admins can use are active?  In those cases, there actually isn't any reason for a non-admin to connect.

**Comment by beaulebens** [comment]
Maybe we can do an informational splash-screen type thing that tells them what's available, but says that they need an admin to enable things? Not entirely sure on that one. @richardmuscat, do you have any ideas/preferences?

**Comment by richardmuscat** [comment]
Seems like one of those things that @MichaelArestad @jeffgolenski @jessefriedman and I should look at over the coming couple weeks?

**Comment by MichaelArestad** [comment]
Some initial thoughts:

> What do we show them when none of the modules that non-admins can use are active?

The simple solution is that we don't show them any of those modules. That means they would likely just have a view with the option to disconnect.

**Comment by MichaelArestad** [comment]
Some initial thoughts:

> What do we show them when none of the modules that non-admins can use are active?

The simple solution is that we don't show them any of those modules. That means they would likely just have a view with the option to disconnect.

The more complex solution would be to show greyed out modules with potentially a call to action to ask the admin to enable it. The active modules should be bubbled to the top, probably.

As far as the landing page, I'm thinking of frankensteining Jeff's Apps view to pitch users on connecting and listing some benefits. 

Once connected, they'll have an At a Glance view with the stats they are allowed to see visible and nothing else. They'll have a settings view for disconnecting, etc.

Thoughts @dereksmart ?

**Comment by MichaelArestad** [comment]
Just to clarify and follow up after a discussion with @jeffgolenski 

At a Glance:
* Hide all cards that a user can't see
* Hide the Upgrades view

Settings:
* grey out settings the user doesn't have access to. allow them to see if active or not (disabled toggle)
* bubble the settings they can tweak to the top of each view.

Pre-connection: We're mocking up a thing, but don't wait on us to make these changes.

**Comment by MichaelArestad** [comment]
@dereksmart Right. We were thinking it might be wise to let them at least the modules available to give them the info needed to ask an admin to enable something. Maybe. Our fallback idea for settings is to just combo the available settings into one list and not show the rest.

We're also thinking of making something like this the connection call to action for non-admins (in progress): https://cloudup.com/cADcM02yie6

**Comment by MichaelArestad** [comment]
@jessefriedman is working on revising the copy here, but this is what we ended up going on the non-admin connection view: 

https://cldup.com/KOqwYxoaTR.jpg

It's similar to the normal connection view, with some tweaks to the copy.

**Comment by richardmuscat** [comment]
Being handled in #4320

**Comment by dereksmart** [comment]
Noticed a few things: 

- Seeing things greyed out as an admin in a couple of the tabs.  
- The stats is showing for an editor and not respecting this option https://cloudup.com/cMAFxfGUlAc
- When protect is disabled, I see an inactive card there prompting me to activate, though I'm not able to.  We should simply hide the card if it's not active.
